django_multitenant/mixins.py

```diff
@@ -59,40 +59,44 @@ def __init__(self, *args, **kwargs):
 
     def __setattr__(self, attrname, val):
 
-        if (attrname in (self.tenant_field, get_tenant_field(self).name)
+        if (
+            attrname in (self.tenant_field, get_tenant_field(self).name)
             and not self._state.adding
             and val
             and self.tenant_value
             and val != self.tenant_value
-            and val != self.tenant_object):
+            and val != self.tenant_object
+        ):
             self._try_update_tenant = True
 
         return super(TenantModelMixin, self).__setattr__(attrname, val)
 
     def _do_update(self, base_qs, using, pk_val, values, update_fields, forced_update):
-        #adding tenant filters for save
-        #Citus requires tenant_id filters for update, hence doing this below change.
+        # adding tenant filters for save
+        # Citus requires tenant_id filters for update, hence doing this below change.
 
         current_tenant = get_current_tenant()
 
         if current_tenant:
             kwargs = get_tenant_filters(self.__class__)
             base_qs = base_qs.filter(**kwargs)
         else:
-            logger.warning('Attempting to update %s instance "%s" without a current tenant '
-                           'set. This may cause issues in a partitioned environment. '
-                           'Recommend calling set_current_tenant() before performing this '
-                           'operation.',
-                           self._meta.model.__name__, self)
-
-        return super(TenantModelMixin,self)._do_update(base_qs, using,
-                                                       pk_val, values,
-                                                       update_fields,
-                                                       forced_update)
+            logger.warning(
+                'Attempting to update %s instance "%s" without a current tenant '
+                "set. This may cause issues in a partitioned environment. "
+                "Recommend calling set_current_tenant() before performing this "
+                "operation.",
```

Comment:
different style of quoting?
doesn't python have the multiline strings, and could we tell the formatter to use those here?

Reply:
difference in quoting style is so there's no need to escape the `"` characters around `%s`. And python does have multiline, but then the resulting string will contain newlines. Which I don't think we want here.